### PR TITLE
Fix EGW upgrade docs, icmp and http probe yaml

### PR DESCRIPTION
**Pull request by sridhartigera**

### CODE REVIEW

**Review by fasaxc — DISMISSED** [review]
LGTM

File: calico-enterprise/networking/egress/egress-gateway-azure.mdx [hunk 1]
```diff
@@ -137,7 +137,6 @@ Azure Route Server deployment is per VNet, as such each VNet with egress gateway
 - [Verify the feature operation](#verify-the-feature-operation)
 - [Controlling the use of egress gateways](#controlling-the-use-of-egress-gateways)
 - [Policy enforcement for flows via an egress gateway](#policy-enforcement-for-flows-via-an-egress-gateway)
-- [Upgrading egress gateways](#upgrading-egress-gateways)
```
Comment (bmckercher123) [inline calico-enterprise/networking/egress/egress-gateway-azure.mdx:140]: List should use active verbs:
Verify
Control
Enforce policy for flows...
Upgrade egress gateways

**Review by bmckercher123 — DISMISSED** [review]
Small section to fix.

File: calico-enterprise/networking/egress/egress-gateway-azure.mdx (hunk 1 above)
Comment (sridhartigera) [reply calico-enterprise/networking/egress/egress-gateway-azure.mdx:140]: FIxed this in the aws and on-prem docs. upgrade is not applicable for azure as this is the first release.

File: calico-enterprise/networking/egress/egress-gateway-aws.mdx [hunk 2]
```diff
@@ -229,6 +229,7 @@ an instance (for example when scaling up the cluster).
 - [Verify the feature operation](#verify-the-feature-operation)
 - [Controlling the use of egress gateways](#controlling-the-use-of-egress-gateways)
```
Comment (bmckercher123) [inline calico-enterprise/networking/egress/egress-gateway-aws.mdx]: Control the use...

**Review by bmckercher123 — DISMISSED** [review]
One more to fix. Approve

**Review by bmckercher123 — APPROVED** [review]
Approve

**Pull request merged**